jrpcfs/api.go

```diff
@@ -165,6 +165,23 @@ import (
 type MountIDAsByteArray [16]byte
 type MountIDAsString string
 
+// BypassReply contains response to BypassReq
+type BypassReply struct {
+	// TODO - update all error vs result vs, etc
+	// want to include Result and Error???
+	ID     uint64 `json:"id"`
+	Result []byte `json:"result"`
```

Comment:
See my similar comment... you "could" do what I did and put an `interface{}` here and, at time of marshaling, assign the field the particular type of the return from the method call you make.